_health-care/_js/components/ProgressButton.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import _ from 'lodash';
+
+/**
+ * A component for the continue button to navigate through panels of questions.
+ *
+ * Required props
+ * `onButtonClick`: function that changes the path to the next panel or submit.
+ * `buttonText`: String. Stores the value for the button text.
+ * `buttonClass`: String. Stores the value for the button class(es).
+ */
+
+class ProgressButton extends React.Component {
+  componentWillMount() {
+    this.id = _.uniqueId();
+  }
+
+  render() {
+    return (
+      <div className="row progress-buttons">
+        <div className="small-8 columns">
```

Comment:
Ah gotcha, that makes sense. I think at least we should put the continue button on top, just to make that more prevalent. And then we can review with the other designers this afternoon to ask their opinion, I am open to either!